app/views/portfolio_evidence_mailer/portfolio_auto_ready.text.erb

```diff
@@ -0,0 +1,13 @@
+Hi <%= @student.first_name %>,
+
+Your portfolio has been automatically compiled as the date for automatic generation has been reached.
```

Comment:
@macite thoughts on the message sent to the user?

Reply:
This should be ok. We need to expand the option to customise this and other emails. So there could be default text, but it could be changed by unit.

Reply:
@macite Good idea, I'll add this to the backlog of tasks for the squad next trimester!